documentation/3_Layers_of_the_Reference_Architecture_Model/3_3_Process_Layer/3_3_5_Publishing_and_using_Data_Apps.md

```diff
@@ -1,8 +1,23 @@
 
-In order to be published, certain Data Apps require certification from the Certification Body (see section 3.5.1) (see first step of the process shown in Figure 3.12). When it comes to using a Data App that is offered by an App Store, App Users (Data Provider or Data Consumer) need to execute a process that is very similar to the 'Exchange Data' process described above.
+On a conceptual level, Data Apps can be treated the same way as Data Offerings in the International Data Spaces (see Section [3.3.2](3_3_2_Data_Offering.md)). Therefore, just as a Data Offering is provided by a Data Provider using an IDS Connector registered at an IDS Metadata Broker, Data Apps are created by an App Provider and registered at an IDS App Store using the App Provider's IDS Connector as a means to communicate with the IDS App Store. As a consequence, App Providers also need to follow the Onboarding Process explained in Section [3.3.1](3_3_1_Onboarding.md). Instead of registering their IDS Connector at an IDS Metadata Broker, App Providers register their Data Apps at an IDS App Store.
 
-For each Data App that was successfully certified, the corresponding metadata is stored in the App Store for being retrieved by users (e.g., Data Consumers or Data Providers) via a search interface. Searching for a Data App is part of the 'Find App' sub process depicted in Figure **//TODO**. If a user finds a suitable Data App (i.e., matching in functionality and compatible with the user's Connector packaging format) in the App Store, the App can be requested. This is indicated in the 'Retrieve App' sub process, which is conceptually identical with the 'Invoke Data Operation' process outlined in section 3.3.2, which is why a detailed discussion is omitted here.
+After a successful Onboarding Process, App Providers can use their IDS Connector to communicate with an IDS App Store and publish a Data App. Alternatively, providers can publish Data Apps to the App Store without the use of an IDS Connector if one is not available. The "Data App Publication Process" is depicted in Figure [3.3.5.1](#PublishingDataApp) below. In order to be published, certain Data Apps require certification from the Certification Body (see [Section 4.2](../../4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective)), as depicted in the first step of the "Data App Publication Process". For each Data App that was successfully certified, the corresponding metadata and app image are stored in the IDS App Store and can be retrieved by IDS Participants via a search interface offered by the App Store.
+
+![PublishingDataApp](./media/data-app-publication-process.png)
```

Comment:
In the figure, the IDS App image can only be published by the user and not by a connector. This is not mentioned in the text. The text only says that you can publish IDS Apps via the connector or alternatively as a user.

Reply:
addressed in commit [00996cd](https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/pull/145/commits/00996cd974b30dd365434dd7bf039a8a9e1257e8)

Reply:
I am not quite sure if the figure is correct/meaningful. I mean it's just a little weird when you upload the app image via the App Store UI, but then upload the metadata via the connector. This is simply a break in the user journey/UI and makes no sense to me.

Reply:
Fixed in commit https://github.com/International-Data-Spaces-Association/IDS-RAM_4_0/pull/145/commits/327ff5ab098939f26cfc74dcc572b2dea23c4679 . I removed the connector's role from the publication process in 3_3_5 and 3_5_3